### PR TITLE
chore(cli-preview): add telemetry for preview command

### PR DESCRIPTION
**Pull request by kuojianlu**
add telemetry for preview command.

### CODE REVIEW

File: packages/cli/src/cmds/preview/cpUtils.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// NOTE:
+// DO NOT EDIT this file in function plugin.
```
Comment (swatDong) [inline packages/cli/src/cmds/preview/cpUtils.ts:6]: Not function plugin...

Comment (kuojianlu) [reply packages/cli/src/cmds/preview/cpUtils.ts:6]: How about just remove these comments?

Comment (swatDong) [reply packages/cli/src/cmds/preview/cpUtils.ts:6]: It depends if you need the copy logic for this file as well

Comment (kuojianlu) [reply packages/cli/src/cmds/preview/cpUtils.ts]: I think not. Just removed them for simplicity.

**Review by swatDong — APPROVED** [review]

File: packages/cli/src/cmds/preview/preview.ts [hunk 2]
```diff
@@ -63,27 +70,44 @@ export default class Preview extends YargsCommand {
...
+      cliTelemetry.sendTelemetryEvent(TelemetryEvent.PreviewStart);
+      const error = errors.ExclusiveLocalRemoteOptions();
+      cliTelemetry.sendTelemetryErrorEvent(TelemetryEvent.Preview, error);
+      return err(error);
     }
 
+    const previewType = args.local || (!args.local && !args.remote) ? "local" : "remote";
+    this.telemetryProperties[TelemetryProperty.PreviewType] = previewType;
+
     const workspaceFolder = path.resolve(args.folder as string);
```
Comment (chagong) [inline packages/cli/src/cmds/preview/preview.ts:82]: You need to enable RootFolder for preview command telemetries, I think. This is to make sure we can get some config context in telemetry like "appid". Do something like this: CliTelemetry.withRootFolder(rootFolder).sendTelemetryEvent(TelemetryEvent.ProvisionStart);

Comment (kuojianlu) [reply packages/cli/src/cmds/preview/preview.ts:82]: Sure. Updated to use `cliTelemetry.withRootFolder`.

**Review by chagong — APPROVED** [review]

**Pull request merged**